heroquotes/__init__.py

```diff
@@ -1,243 +1,15 @@
-
-		elif val == 48 :
-			print("That’s my secret, Captain. I’m always angry. - Bruce Banner")
-
-		elif val == 49 :
-			print("We have a Hulk. - Tony Stark")
-
-
-	elif val >= 50 and val <= 74 :
-
-		if val == 50 :
-			print("You get hurt, hurt 'em back. You get killed… walk it off. - Captain America")
-
-		elif val == 51 :
-			print("Sir, I’m going to have to ask you to exit the donut. - Nick Fury")
-
-		elif val == 52 :
-			print("I told you, I don't want to join your super-secret boy band. - Iron Man")
-
-		elif val == 53 :
-			print("I’m just a kid from Brooklyn. - Steve Rogers")
-
-		elif val == 54 :
-			print("Genius. Billionaire. Playboy. Philanthropist. - Tony Stark")
-
-		elif val == 55 :
-			print("Let me know if ‘real power' wants a magazine or anything. - Nick Fury")
-
-		elif val == 56 :
-			print("There’s only one God, ma’am, and I’m pretty sure he doesn’t dress like that. - Steve Rogers")
-
-		elif val == 57 :
-			print("He’s adopted. - Thor")
-
-		elif val == 58 :
-			print("Shakespeare in the park? Doth mother know you weareth her drapes? - Tony Stark")
-
-		elif val == 59 :
-			print("It’s good to meet you, Dr. Banner.\nYour work on anti-electron collisions is unparalleled.\nAnd I’m a huge fan of the way you lose control and turn into an enormous green rage monster. - Tony Stark")
-
-		elif val == 60 :
-			print("No hard feeling Point Break, you've got a mean swing. - Tony Stark")
-
-		elif val == 61 :
-			print("As always sir, a great pleasure watching you work. - Jarvis")
-
-		elif val == 62 :
-			print("Before we get started, does anyone want to get out? - Steve Rogers")
-
-		elif val == 63 :
-			print("On your left. - Steve Rogers")
-
-		elif val == 64 :
-			print("I do what he does, just slower. - Sam Wilson")
-
-		elif val == 65 :
-			print("The city is flying. We’re fighting an army of robots.\nAnd I have a bow and arrow. None of this makes sense. - Clint Barton")
-
-		elif val == 66 :
-			print("Ah, she left me. And my mom died too.\nAnd my dad got deported. But I got the van! - Luis")
-
-		elif val == 67 :
-			print("So, you like cats? - Sam Wilson")
-
-		elif val == 68 :
-			print("Anybody on our side hiding any shocking and fantastic abilities they’d like to disclose?\nI’m open to suggestions. - Tony Stark")
-
-		elif val == 69 :
-			print("I don't know if you've been in a fight before but there's usually not this much talking. - Sam Wilson")
-
-		elif val == 70 :
-			print("Just a typical homecoming, on the outside of an invisible jet, fighting my girlfriend's dad. - Spiderman")
-
-		elif val == 71 :
-			print("Dude, you’re embarrassing me in front of the wizards. - Tony Stark")
-
-		elif val == 72 :
-			print("I'm with you 'til the end of the line. - Steve Rogers")
-
-		elif val == 73 :
-			print("I love you 3000. - Tony Stark")
-
-		elif val == 74 :
-			print("When you look in her eyes and she’s looking back in yours…everything…feels…not quite normal. - Peter Parker")	
+    return quotes_dict[randint(0, 74)]
```

Comment:
returns the respective item associated with the randomly generated key inclusive (0,74) from quotes_dict Dictionary in quotes.py